webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends "@DOMJudge/jury/base.html.twig" %}
+
```

Comment:
If you add

```twig
{% import "@DOMJudge/jury/jury_macros.twig" as macros %}

{% block extrahead %}
    {{ parent() }}
    {{ macros.select2_extrahead() }}
{% endblock %}
```

you get the Select2-based multi-selects, which IMHO look nicer (and we also have in some other places like judgehost restrictions)

Reply:
beautiful, thanks!